tools/create_release_notes.py

```diff
@@ -34,6 +35,7 @@
 RELEASE_FILE_CONTENT_TEMPLATE = """\
 ---
 title: OpenPOWER Host OS release
+release_tag: {release_tag}
```

Comment:
I'll keep "tag". It should match the git tag to be created for versions git repository.